app/views/podcasts/_form_main.html.erb

```diff
@@ -8,9 +8,11 @@
 
 <div class="col-12 mb-4">
   <div class="form-floating input-group">
-    <%= form.text_field :subtitle %>
-    <%= form.label :subtitle %>
-    <%= field_help_text t(".help.subtitle") %>
+    <%= form.fields_for :default_feed do |fields| %>
```

Comment:
Moved these proxied fields to actually be form fields on the default_feed.  Seems less magical this way.

No other changes to this file.